WordPress/Classes/Utility/Analytics/WPAnalyics+Logging.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+import Pulse
+import WordPressShared
+
+final class WPAnalyticsLogger: NSObject, WPAnalyticsTracker {
```

Comment:
This class name feels too broad to me. Since this implementation is used in the Debug Menu specifically, what do you think renaming it to be more explicit around debugging?

Reply:
That's a good idea; it's not clear from the name. I changed it to `DebugAnalyticsLogger`, which is a common prefix for all the debug features.